interapp-frontend/src/components/AnnouncementList/AnnouncementList.tsx

```diff
@@ -0,0 +1,17 @@
+import { Text, Title } from '@mantine/core';
+import './styles.css';
+
+export default function AnnouncementList(props: {number_of_announcements: number; className?: any}) {
```

Comment:
pls dont use ``any`` for className, use ``string`` instead.
https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#any 

also please format your code with prettier ``bun run prettier`` in ``/interapp-frontend``